analyzers/src/SonarAnalyzer.CSharp/Rules/CertificateValidationCheck.cs

```diff
@@ -123,5 +123,9 @@ protected override ImmutableArray<Location> LambdaLocations(InspectionContext c,
             reference.GetSyntax();   // VB.NET has more complicated logic
 
         private protected override KnownType GenericDelegateType() => KnownType.System_Func_T1_T2_T3_T4_TResult;
+
+        private protected override bool AreSameMethods(InvocationExpressionSyntax invocation, string methodName) =>
```

Comment:
This should be doable without an abstract method. Either with `node.ToStringContains(...)` or even better `Language.Syntax.InvocationIdentifier()`